inst/stan/functions/rt.stan

```diff
@@ -52,16 +52,98 @@ vector update_Rt(int t, real R0, vector noise, array[] int bps,
  * @param bp_effects Vector of breakpoint effects
  * @param bp_sd Array of breakpoint standard deviations
  * @param bp_n Number of breakpoints
- * @param prior_infections Prior mean for initial infections
  */
 void rt_lp(array[] real initial_infections, vector bp_effects,
-           array[] real bp_sd, int bp_n, real prior_infections) {
+           array[] real bp_sd, int bp_n, array[] int cases) {
   //breakpoint effects on Rt
   if (bp_n > 0) {
     bp_sd[1] ~ normal(0, 0.1) T[0,];
     bp_effects ~ normal(0, bp_sd[1]);
   }
-  // initial infections
-  initial_infections ~ normal(prior_infections, sqrt(prior_infections));
-  
+  // initial infections scaling (on the log scale)
+  int head_count = num_elements(cases) > 7 ? 7 : num_elements(cases);
+  real infections_guess = fmax(1, mean(head(cases, head_count)));
+  initial_infections ~ normal(log(infections_guess), 2);
+}
+
+/**
+ * Calculate the negative moment generating function (nMGF) of a probability
```

Comment:
we should really give sam some attribution in here

Reply:
if he could contribute some maths so support in the vignette that would be cool @samuelbrand1

Reply:
I've linked out to the original code now - contributions to the methodology docs welcome!